### PR TITLE
fixed the sort method by adding params

### PR DESCRIPTION
**Pull request by devSabz404**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -477,7 +477,7 @@
   // Fill version selector.
   let select = $("#select_version");
   let allVersions = [... (new Set(versions.map(v => parseInt(v[1])))).values()]
-                      .sort()
+                      .sort((a,b)=>a-b)
```
Comment (mimi89999) [inline index.html:480]: That's not according to the code style. In https://github.com/mozilla/usecounters/blob/gh-pages/index.html#L354 there are spaces between argument definitions and also between `=>`
There are also spaces between math operators and operands:
https://github.com/mozilla/usecounters/blob/gh-pages/index.html#L312
https://github.com/mozilla/usecounters/blob/gh-pages/index.html#L391

**Review by mimi89999 — CHANGES_REQUESTED** [review]
I think that it would also be good to reference the bug in the commit message

Anyway, I already opened https://github.com/mozilla/usecounters/pull/2/ before.

File: index.html (hunk 1 above)
Comment (devSabz404) [reply index.html:480]: thanks will make the necessary changes

**Comment by mimi89999** [comment]
I think that this PR can be closed since https://github.com/mozilla/usecounters/pull/2 was merged.

**Pull request closed without merging**